src/main/java/com/denizenscript/depenizen/bukkit/properties/superiorskyblock/SuperiorSkyblockPlayerExtensions.java

```diff
@@ -0,0 +1,134 @@
+package com.denizenscript.depenizen.bukkit.properties.superiorskyblock;
+
+import com.bgsoftware.superiorskyblock.api.SuperiorSkyblockAPI;
+import com.bgsoftware.superiorskyblock.api.island.Island;
+import com.bgsoftware.superiorskyblock.api.island.PlayerRole;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.depenizen.bukkit.objects.superiorskyblock.SuperiorSkyblockIslandTag;
+
+public class SuperiorSkyblockPlayerExtensions {
+
+    public SuperiorSkyblockPlayerExtensions(PlayerTag player) {
+        this.player = player;
+    }
+
+    PlayerTag player;
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <PlayerTag.superiorskyblock_bypass_mode>
+        // @returns ElementTag(Boolean)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @mechanism PlayerTag.superiorskyblock_bypass_mode
+        // @description
+        // Returns whether a player can build on islands that they are not a part of.
+        // -->
+        PlayerTag.tagProcessor.registerTag(ElementTag.class, "superiorskyblock_bypass_mode", (attribute, player) -> {
+            return new ElementTag(SuperiorSkyblockAPI.getPlayer(player.getUUID()).hasBypassModeEnabled());
```

Comment:
could probably make a helper method for `SuperiorSkyblockAPI.getPlayer(player.getUUID())`? it is repeated quite a bit